src/core/EdFi.DataManagementService.Core/ApiSchema/ApiSchemaSchemaProvider.cs

```diff
@@ -29,7 +31,7 @@ public class ApiSchemaSchemaProvider(ILogger<ApiSchemaSchemaProvider> _logger) :
             _logger.LogDebug("Entering ApiSchemaSchemaProvider");
 
             string schemaContent = File.ReadAllText(
-                Path.Combine("Core", "ApiSchema", "ApiSchema_Schema.json")
+                Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) ?? "", "ApiSchema", "ApiSchema_Schema.json")
```

Comment:
I wonder if we should embed this schema file in the DLL, instead of having a plain text file that must be copied around?

Not in the scope of this ticket though.